tests/main/hidden-snap-dir/task.yaml

```
@@ -0,0 +1,76 @@
+summary: Check that the experimental hidden dir feature migrates the dir
+
+environment:
+    NAME: test-snapd-tools
+    FILE: test-snapd-tools_1.0_all.snap
+
+prepare: |
+    snap pack "$TESTSLIB"/snaps/"$NAME"
+    snap install --dangerous ./"$FILE"
```

this should work too:
```suggestion
    "$TESTSTOOLS"/snaps-state install-local "$NAME"
```